fs/copy.py

```diff
@@ -54,100 +53,168 @@ def copy_fs_if_newer(
     workers=0,  # type: int
 ):
     # type: (...) -> None
-    """Copy the contents of one filesystem to another, checking times.
+    """Use ``copy_fs_if`` instead."""
```

Comment:
For documenting deprecation, please keep the previous summary, and add the [`deprecated`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) Sphinx directive:
```python
"""Copy the contents of one filesystem to another, checking times.

.. deprecated:: 2.5.0
   Use `~fs.copy_fs_if` with ``condition="newer"`` instead.
"""
```